watertap/property_models/activated_sludge/tests/test_asm1_integration.py

```diff
@@ -0,0 +1,216 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Tests for ASM1 proeprty and reaction packages.
+
+Verified againts results from:
+
+[1] J. Alex, L. Benedetti, J. Copp, K.V. Gernaey, U. Jeppsson, I. Nopens, M.N. Pons,
+J.P. Steyer and P. Vanrolleghem, "Benchmark Simulation Model no. 1 (BSM1)", 2018
+"""
+
+# Some more inforation about this module
+__author__ = "Andrew Lee"
+
+import pytest
+
+import pyomo.environ as pyo
+from pyomo.network import Arc
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.models.unit_models import CSTR
```

Comment:
```suggestion
from idaes.generic_models.unit_models import CSTR
```

So, it looks like we need to update our fork of IDAES and the tag that gets installed. The 'idaes.models' import statement does not currently work in our repo.

Reply:
There are deprecation paths in place for now, but these changes  will need to be made at some point.

Reply:
Opened an issue (#503) as a reminder for us.